test/integration/testdata/test-docker-build.json

```diff
@@ -0,0 +1,29 @@
+{
+  "kind":"BuildConfig",
```

Comment:
1) there were no existing buildconfig definitions you could use for these 2 things?
2) let's start creating a builds subdir in testdata if we are going to add these new ones.

Reply:
Updated (moved to builds dir)

Reply:
Updated (used existing json files)